src/stories/Users/VerifyAttributeWithLink/story.js

```diff
@@ -0,0 +1,45 @@
+const usersRepo = requireRepo("users");
+
+
+const prepare = ({ reqQuery, reqBody, reqParams, req }) => {
+  const payload = reqParams
+  return payload;
+};
+
+const authorize = async ({ prepareResult }) => {
+  try {
+    if (0) {
+      throw {
+        statusCode: 401,
+        message: "Unauthorized",
+      };
+    }
+
+    return true;
+  } catch (error) {
+    throw error;
+  }
+};
+
+const handle = async ({ prepareResult, authorizeResult }) => {
+  try {
+    return await usersRepo.verifyAttributesWithLink(prepareResult);
+  } catch (error) {
+    throw error;
+  }
+};
+
+const respond = async ({ handleResult }) => {
+  try {
+    return handleResult;
```

Comment:
We are not doing the redirection part yet, though you have logged it here: process.env.SUCCESS_REDIRECT_URL

As we discussed, we should send these redirect_urls are part of the payload, so that URL can be built in such a way.

Reply:
@rjvim if we want to pass the redirect redirect_url as part of the payload  then it is better to pass them along with the registration payload so that it can prevent too much interaction with access, this is my thought